src/pages/Home.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Grid,
+  Map,
+  AppBar,
+  Search,
+  Drawer,
+  ListCard,
+} from '@digicatapult/ui-component-library'
+
+import Logo from '../assets/images/hii-logo.png'
+import geojson from '../assets/hii.json'
+
+const HomeBar = styled.div`
+  height: 100%;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-image: url(${Logo});
+  margin-left: 20px;
+`
+
+const Sidebar = styled.div`
+  display: grid;
+  height: 100%;
+  grid-template-rows: auto auto 1fr;
+`
+
+const ListWrapper = styled.div`
+  display: grid;
+  gap: 5px;
+  font-size: 1em;
+  overflow: scroll;
+  padding: 5px 0 0 5px;
+`
+
+const SearchWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  font-size: 1rem;
+  background: #216968;
+`
+
+export default function Home() {
+  return (
+    <Grid
+      areas={[
+        ['home', 'header', 'header'],
+        ['sidebar', 'main', 'main'],
+      ]}
+      columns={['minmax(10%, 20%)', '1fr', '1fr']}
+      rows={['80px', '1000px']} //TODO make 2nd row use rest of vh
```

Comment:
three ways to do this:
1. `rows={['80px', 'calc(100vh-80px)']}`
2. style the outer grid here by creating a styled component here. So a `styled(Grid)`. You can set the width and height of the container to be `100vw`/`100vh`
3. Implement `width` and `height` props on the `Grid` component in the component lib and then update here